packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -13,117 +13,104 @@ import {
 	SelectControl,
 	TextControl,
 } from '@wordpress/components';
-import { Component } from '@wordpress/element';
 
-class ImageSizeControl extends Component {
-	/**
-	 * Run additional operations during component initialization.
-	 *
-	 * @param {Object} props
-	 */
-	constructor( props ) {
-		super( props );
-
-		this.updateDimensions = this.updateDimensions.bind( this );
-	}
-
-	updateDimensions( width = undefined, height = undefined ) {
+const ImageSizeControl = ( {
+	imageWidth,
+	imageHeight,
+	imageSizeOptions = [],
+	isResizable = true,
+	slug,
+	width,
+	height,
+	onChange,
+	onChangeImage = noop,
+} ) => {
+	const updateDimensions = (
+		nextWidth = undefined,
+		nextHeight = undefined
```

Comment:
Parameters default to `undefined` anyway, so the default parameters here are unnecessary.

Reply:
Thanks Reviewing ! fixed now.